xbmc/ApplicationStackHelper.cpp

```diff
@@ -283,3 +283,15 @@ CApplicationStackHelper::StackPartInformationPtr CApplicationStackHelper::GetSta
   }
   return m_stackmap[key];
 }
+
+CApplicationStackHelper::StackPartInformationPtr CApplicationStackHelper::GetStackPartInformation(
+    const std::string& key) const
+{
+  const auto it = m_stackmap.find(key);
+  if (it == m_stackmap.end())
```

Comment:
Use `std::none_of`?

Reply:
I don't see how none_of would make this any clearer. it would mean keeping the unnecessary double find that you point out.